examples/cosmos-sdk/transfer.md

```diff
@@ -0,0 +1,243 @@
+```
+
+### Install Atomkraft
+
+```
+pip install --upgrade atomkraft
+```
+
+### Initialize project
+
+```
+atomkraft init transfer
+cd transfer
+```
+
+### Turn on Poetry shell
+
+```
+poetry shell
+```
+
+### Create a model specification
+
+We will model a simple token transfer between two users.
+Alice and Bob both start with 100 tokens.
+At each step, Alice or Bob will send some amount of tokens to the other person.
+We will use TLA+ to specify this model.
+You can use the following code to _jump-start_ a new model at `models/transfer.tla`.
+
+```
+cat > models/transfer.tla << EOF
+---- MODULE transfer ----
+EXTENDS Apalache, Variants, Integers, FiniteSets
+
+VARIABLES
+    \* @type: Int -> Int;
+    balances,
+    \* @type: Init({n_wallet: Int}) | Transfer({sender: Int, receiver: Int, amount: Int});
+    action,
+    \* @type: Int;
+    step
+
+WALLETS == 0..1
+
+Init ==
+    /\ balances = [wallet \in WALLETS |-> 100]
+    /\ action = Variant("Init", [n_wallet |-> Cardinality(WALLETS)])
+    /\ step = 0
+
+Next ==
+    \E sender \in WALLETS:
+    \E receiver \in WALLETS:
+    \E amount \in 0..balances[sender]:
+        /\ sender /= receiver
+        /\ balances' = [
+            balances EXCEPT
+            ![sender] = @ - amount,
+            ![receiver] = @ + amount
+            ]
+        /\ action' = Variant("Transfer", [sender |-> sender, receiver |-> receiver, amount |-> amount])
+    /\ step' = step + 1
+
+View ==
+    IF VariantTag(action) = "Transfer"
+    THEN VariantGetUnsafe("Transfer", action)
+    ELSE [sender |-> -1, receiver |-> -1, amount |-> 0]
+
+Inv == step < 10
+
+====
+EOF
+```
+
+### Generate traces
+
+#### Download Apalache
+
+We will use Apalache to generate traces from our model.
+
+```
+curl -Lo- https://github.com/informalsystems/apalache/releases/download/v0.26.0/apalache-0.26.0.tgz | tar -zxf-
+```
+
+The Apalache executable will be at `./apalache-0.26.0/bin/apalache-mc`.
+
+The following will generate traces at `traces/`.
+
+```
+./apalache-0.26.0/bin/apalache-mc check --features=rows --init=Init --next=Next --inv=Inv --view=View --max-error=10 --run-dir=mc_traces models/transfer.tla
+ls -1I violation.itf.json mc_traces/violation*itf.json | xargs cp -t traces
+rm -r mc_traces
+```
+
+### Generate reactors
+
+Once we have some traces, we can generate reactor stubs for the traces.
+
+In our model, the `action` variable had two tags - `Init`, `Transfer`.
+
+```
+atomkraft reactor --actions "Init,Transfer" --variables "action"
+```
+
+### Setting up chains
+
+Once we have the stub ready, we can set up the chain binary for the testnet.
+
+We will use `juno` chain binary. But any Cosmos-SDK derived chain should work.
+
+#### Binary
+
+```
+git clone --depth 1 --branch v9.0.0 https://github.com/CosmosContracts/juno
+(cd juno; make build)
+```
+
+The binary will be at `./juno/bin/junod`
+
+### Chain parameters
+
+Now we can update the chain parameters.
+
+```
+atomkraft chain config chain_id test-cw
```

Comment:
We need to be able to run a single `atomkraft chain config` command on multiple key-values. And the command should output some feedback. This is a comment for a separate issue, not for this PR.